copier.yml

```diff
@@ -1,16 +1,20 @@
 _message_before_copy: |
-  This template will create a new repository which describes the deployment IOCs and
-  services to a Kubernetes cluster using Argocd. Each repository gathers a collection
-  of ec services as Argocd Apps which are deployed to *the same Argocd Project*.
+  This template will create a new repository which describes the set of IOCs and services that Argo CD will track on a Kubernetes cluster.
 
 _subdirectory: "template"
 
 _preserve_symlinks: true
 
 ioc_group:
   type: str
-  help: Short name for the collection of services.
-  placeholder: e.g. "t01", "VA", "PS"
+  help: |
+    A name for the group of IOC and service instances that this
+       repository will track.
+
+       At DLS this should be the short beamline name or the technical area
```

Comment:
When I made this template I avoided any "At DLS..." because I feel that discourages collaboration (suggests other facilities should be forking). My personal opinion is I would discourage becoming "Diamond Epics Containers".

There's definitely a trade off to be made because it seems to be confusing to onboard people more familiar with Diamond then they are of Containerisation. Is it an idea to move this repo rather to the DLS organisation and then make it very specific to our application?

Reply:
lets discuss the fully DLS vs fully generic vs somewhere in between issue in the main thread.